### PR TITLE
Use buttonv1 in contextualmenu

### PR DESCRIPTION
**Pull request by lyzhan7**
Updated import + updated to pass content as children instead since content prop is deprecated

### Platforms Impacted
- [ ] iOS
- [X] macOS
- [ ] win32 (Office)
- [ ] windows
- [ ] android

### Description of changes

Updated ContextualMenu to use ButtonV1, and updated the buttons to pass content as children instead since content prop is deprecated

### Verification

Manual: checked that the buttons looked updated

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
|Screen Shot 2022-04-28 at 12 59 19 PM|Screen Shot 2022-04-28 at 1 05 49 PM
 |

### Pull request checklist

This PR has considered (when applicable):
- [ ] Automated Tests
- [ ] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

**Review by rurikoaraki — APPROVED** [review]

**Comment by lyzhan7** [comment]
I think I need to run yarn change to fix the failures, but for some reason when I run it nothing happens (I'm expecting a commit to be made titled "Change Files" but I don't see anything).
Screen Shot 2022-04-28 at 2 48 02 PM

**Comment by Saadnajmi** [comment]
@lyzhan7 Try making sure your fork's master is up to date with upstream master. One non-intuitive thing I learned about beachball is that it compares your changes against whatever `origin/master` is. Because your local clone specifies origin as "your fork", while the CI specifies origin as "the main microsoft repo", when those two are out of sync, `yarn change` can behave differently between your local machine and the CI.

Tl;dr: Try pressing the "Fetch upstream" button on https://github.com/lyzhan7/fluentui-react-native

**Comment by lyzhan7** [comment]
Thanks Saad, will try to keep that in mind. And thanks for helping me figure out why my git branches were in a weird state and figuring out the change file!

**Review by Saadnajmi — APPROVED** [review]
